bitrise.yml

```diff
@@ -1,19 +1,13 @@
-format_version: 1.0.0
```

Comment:
The project's CI setup has been changed recently this change pulls the bitrise.yml from the master branch.